Menu/Menu.js

```diff
@@ -1,10 +1,27 @@
 
 const toggleMenu = () => {
-  // Toggle the "menu--open" class on your menu refence. 
+  // Toggle the "menu--open" class on your menu refence.
+  menu.classList.toggle('menu--open');
 }
 
 // Start Here: Create a reference to the ".menu" class
-const menu;
+const menu = document.getElementsByClassName('menu')[0];
+
+class menuitem
+{
+  constructor(element)
+  {
+    this.element = element;
+    this.element.addEventListener("click", x=> this.clicked())
+  }
+  clicked()
+  {
+    console.log("clicked");
+  }
+}
 // create a reference to the ".menu-button" class
-const menuButton;
+const menuButton = document.getElementsByClassName('menu-button')[0];
 // Using your menuButton reference, add a click handler that calls toggleMenu
+menuButton.addEventListener("click", function(){
+  toggleMenu();
+});
```

Comment:
You don't have to invoke `toggleMenu` inside an anonymous function in this case - simply passing `toggleMenu` as a callback argument will do.